dev/benchmarks/generate_fixtures.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Note that, unless Nanoarrow IPC doesn't support writing, this might be done directly in the benchmarking setup.